packages/tests/backend/sirius-components-tests/src/test/java/org/eclipse/sirius/components/tests/GeneralPurposeTests.java

```diff
@@ -171,8 +171,8 @@ private List<Path> findFilePaths(Path sourceFolderPath, String extension) {
     public void checkJavaCode() {
         File rootFolder = this.getRootFolder();
         Path backendFolderPath = Paths.get(rootFolder.getAbsolutePath(), BACKEND_FOLDER_PATH);
-        List<Path> javaFilePaths = this.findFilePaths(backendFolderPath, JAVA_FILE_EXTENSION);
-        for (Path javaFilePath : javaFilePaths) {
+        List<Path> javaFilePathsWithoutGenerated = this.findFilePaths(backendFolderPath, JAVA_FILE_EXTENSION, true);
```

Comment:
Don't scan and read every single line of every single Java files in the project twice. Put the filter to select the tests to perform `path.contains("/sirius-components-domain"))` inside of the for loop performing the various tests

Reply:
Done